data/sebashmh.json

```
@@ -0,0 +1,57 @@
+{
+  "name": "Sebastian Henao Madrigal",
+  "type": "personal",
+  "bio": "Computer Scientist, Engineering Manager, FullStack Engineer, Tech entrepreneur, Travel enthusiast, Photographer & Coffee Nerd",
+  "tags": [
+    "Open Source",
+    "JavaScript",
+    "Python",
+    "DevOps",
+    "Kubernetes",
+    "Docker",
+    "Node.js",
+    "AWS"
+  ],
+    "socials": [
+    { "icon": "FaTwitter", "url": "https://twitter.com/madriantor" },
+    { "icon": "FaGithub", "url": "https://github.com/sebashmh" },
+    { "icon": "FaLinkedin", "url": "https://www.linkedin.com/in/sebastianhenaomadrigal/" },
+    { "icon": "FaInstagram", "url": "https://vero.co/sebasmadrigal" },
+    { "icon": "FaLink", "url": "https://sebas.io" },
```

There is an extra comma leading json check fail. 

```suggestion
    { "icon": "FaLink", "url": "https://sebas.io" }
```

---

Comma removed, thanks.